composer.json

```diff
@@ -18,16 +18,19 @@
     "php": "^7.4 || ^8",
     "ext-calendar": "*",
     "illuminate/support": "^8.73 || ^9 || ^10",
-    "mll-lab/microplate": "^6",
     "mll-lab/str_putcsv": "^1",
     "nesbot/carbon": "^2.62.1",
     "thecodingmachine/safe": "^1 || ^2"
   },
   "require-dev": {
     "ergebnis/composer-normalize": "^2",
+    "illuminate/database": "^8.73 || ^9 || ^10",
```

Comment:
I don't think it is necessary to add this as a direct dependency or suggest it, as it is only needed for the model casts. So everybody who needs the casts will already have it.